pillar/params.sls

```diff
@@ -26,6 +26,12 @@ kube_group:       'kube'
 # install the addons (ie, DNS)
 addons:           'false'
 
+ssl:
+  enabled:        true
+  ca_file:        '/etc/pki/trust/anchors/SUSE_CaaSP_CA.crt'
+  crt_file:       '/etc/pki/minion.crt'
+  key_file:       '/etc/pki/minion.key'
+
```

Comment:
This adds some redundancy/inconsistency in the Pillar: see line 29: there is `paths:ca_dir` and `paths:ca_filename`